sdk/python/feast/type_map.py

```diff
@@ -326,7 +328,14 @@ def _python_value_to_proto_value(
                     ProtoValue(unix_timestamp_list_val=Int64List(val=ts))  # type: ignore
                     for ts in int_timestamps_lists
                 ]
-
+            if feast_value_type == ValueType.BOOL_LIST:
+                # ProtoValue no longer supports conversion of np.bool_ so we need it convert it.
```

Comment:
did it support it at some point?

Reply:
I'm not quite sure, I couldn't find any real information on it but it was in the codebase and was working, ProtoValue just doesn't work on np.bool_ now.

Reply:
Got it. I'd update the comment that it's to support np.bool_ instead then

Reply:
Done.